substrate-node/pallets/pallet-tft-price/src/tests.rs

```diff
@@ -222,6 +223,16 @@ fn test_parse_price() {
     })
 }
 
+#[test]
```

Comment:
Would be good to add a test that tries to parse a wrong json request.

Reply:
added test for empty and incomplete requests